openstates/in/utils.py

```diff
@@ -0,0 +1,22 @@
+import requests
+
+
+def get_with_increasing_timeout(scraper,link,fail=False,kwargs={}):
```

Comment:
Does a scrapelib.Session() give me the option to retry a lot like this? I want to be able to hit them with increasingly long timeouts.